src/app/containers/FrontPageMain/index.jsx

```diff
@@ -1,16 +1,28 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useContext } from 'react';
 import { shape } from 'prop-types';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { frontPageDataPropTypes } from '../../models/propTypes/frontPage';
 import { GhostWrapper, GridItemConstrainedLarge } from '../../lib/styledGrid';
+import { ServiceContext } from '../../contexts/ServiceContext';
 import FrontPageSection from '../FrontPageSection';
 import deepGet from '../../helpers/json/deepGet';
 
 const FrontPageMain = ({ frontPageData }) => {
+  const { serviceLocalizedName, translations } = useContext(ServiceContext);
+  const { home } = translations;
+
   const groups = deepGet(['content', 'groups'], frontPageData);
 
+  const offScreenText = (
+    <Fragment>
+      <span lang="en-GB">BBC News</span>, {serviceLocalizedName} - {home}
```

Comment:
Sorry to nit pick, but Simorgh isnt JUST BBC News technically, we should account for the possibility of 1 day doing a non BBC News service, can you import BBC News from config please?

Reply:
Ok, but we don't have just `BBC News` in the config, which means we have to add another key:value and it is becoming messy.

Reply:
Also, how should we named the key, since we want to make it generic and `brandName` is already taken?

Reply:
Also, how should we name the key, since we want to make it generic and `brandName` is already taken?

Reply:
Erm, `product` maybe? it doesnt really have a name, so any we pick will be wrong :P